File: husky/app/src/test/java/com/keylesspalace/tusky/util/ViewPager2FixTest.java
package com.keylesspalace.tusky.util;

import java.lang.reflect.Field;
import org.junit.Assert;
import org.junit.Test;

public class ViewPager2FixTest {

    @Test
    public void getViewPagerRecyclerViewFieldTest() {
        try {
            Field f = ViewPager2Fix.getViewPagerRecyclerViewField();
        } catch(Exception e) {
            Assert.fail("asdf");
        }
    }

    @Test
    public void getRecyclerViewTouchSlopFieldTest() {
        try {
            Field f = ViewPager2Fix.getRecyclerViewTouchSlopField();
        } catch(Exception e) {
            Assert.fail("asdf");
        }
    }
}

File: husky/app/src/test/java/com/keylesspalace/tusky/util/ViewPager2FixTest.kt
package com.keylesspalace.tusky.util

import org.junit.Assert
import org.junit.Test

class ViewPager2FixTest {

    @Test
    fun viewPagerRecyclerViewFieldTest() {
        try {
            val f = ViewPager2Fix.getViewPagerRecyclerViewField()
        } catch (e: Exception) {
            Assert.fail("asdf")
        }
    }

    @Test
    fun recyclerViewTouchSlopFieldTest() {
        try {
            val f = ViewPager2Fix.getRecyclerViewTouchSlopField()
        } catch (e: Exception) {
            Assert.fail("asdf")
        }
    }
}
